bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -348,13 +341,10 @@ private void saveToMemoryAndDiskIfNecessary(@NonNull String uuid,
 
     private void writeToDisk(@NonNull String uuid,
                              @NonNull Bundle bundle) {
-        Parcel parcel = Parcel.obtain();
-        parcel.writeBundle(bundle);
-        String encodedString = Base64.encodeToString(parcel.marshall(), 0);
+        String encodedString = BundleUtil.toEncodedString(bundle);
         mSharedPreferences.edit()
                 .putString(getKeyForEncodedBundle(uuid), encodedString)
                 .apply();
-        parcel.recycle();
     }
 
 }
```

Comment:
Wanna toss a newline here

Reply:
Sure, I guess while I'm here I can do that.

Reply:
Done.